### DISCUSSION TITLE
How to make opponent's notes glow?

### DISCUSSION THREAD

**Discussion by Jodda368** [KadeArchive/Kade-Engine#1225] [Q&A] [answered]
I saw a lot newer mods code the opponents notes to glow when being hit, like Tabi and Agoti and i really want to learn it so please help me with like a tutorial or something.

**Comment by isakube** [comment, marked as answer]
you have an option disabled in Appearance, specifically Light CPU Strums

**Reply by Jodda368** [reply]
thank you!
